og.module

```diff
@@ -86,6 +86,16 @@ define('OG_MEMBERSHIP_REQUEST_FIELD', 'og_membership_request');
  */
 
 /**
+* Implements hook_entity_field_storage_info_alter().
+*/
+function og_entity_field_storage_info_alter(&$fields, \Drupal\Core\Entity\EntityTypeInterface $entity_type) {
+  if (!empty($fields[OG_AUDIENCE_FIELD])) {
```

Comment:
Yes, sure. I was just going with the simplest implementation to start with, as I was just concentrating on the storage (none) aspect. Is it OK if I expand that in a follow up? As we might want to think about how we do that - because we need to modify the field definitions that current `_og_is_group_audience_field ` function would mean we would have to iterate every field definition of every entity and ask for the type. That might be ok, but could add a bit of additional overhead. On the plus side, we can now simplify that function even more, as we have an og_membership_reference field type now. So we only need to check that :)